.scalafmt.conf

```diff
@@ -1,3 +1,3 @@
-version = "2.3.0"
+version = "2.7.1"
```

Comment:
What do you think about including scalafmt checks on circleci?

Reply:
At some point, we need to do that. But I prefer to do that in separate PR